website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,288 @@
+
+:::
+
+### Add some order
+
+There's a lot of data there. Dozens of customers are buying hundreds of products. You will sort it so that the customers are listed ascending by their CUSTOMER_ID number, with the most purchased products listed in descending order. 
+
+1. Drag the **Order** operator over to the right of the **Aggregation** tile and connec them.
+2. Click the **pencil edit icon**.
+3. In the **Sort order** field click **Select column** and click `Aggregation1.CUSTOMER_ID` from the dropdown. Set it to `Asc`. 
+4. Click **Add sorting** and in the new **Select column** field select `Aggregation1.count_PRODUCT_ID`. Set it to `Desc`.
+5. Press the **Play icon** to preview the new data.
+
+    <Lightbox src="/img/docs/dbt-cloud/visual-editor/order.png" width="90%" title="The ordered data operator tile config and data preview." />
+
+:::tip
+
+Want to practice on your own? Try adding a **Filter** operator that removes items with less than 10 sales for any customer ID. Be sure to run the preview and verify the data is correct.
+
+:::
+
+## Configure your output model
+
+Now that you've built your model, you need to customize the output name and location:
+1. Drag the **Output Model** operator to the right of your **Order** operator. 
+2. Connect the **Order** and **Output Model** operators.
+3. The **Output Model** configuration will default to the name of your Visual Editor project and the default models directory. Click the **pencil edit icon** to configure the optional fields:
+    - Edit the **Model name** field if you want the name to be different than that of your project.
+    - Edit the **File path** if you have a custom directory for your Visual Editor models. 
+    - Hover over a column name and click the **-** icon to remove it from the output model.
+4. Click the **play icon** to preview your final model. 
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/output-model.png" width="90%" title="The output model configures your final model's name and location." />
+
+:::tip Model locations
+
+You can customize the location for Visual Editor models to keep them separate from other dbt Cloud models. Check with your dbt admins for best practices and ideas for Visual Editor model locations and naming conventions.
+
+:::
+
+## Run and share your model
+
+Now that you've built a model that results in the data you want, it's time to run it and push it to your Git repo. Before you run your model, keep a few items in mind:
+
+- When you run previews (at any stage in the process), it does not affect the state of your warehouse. So, you can test and develop in the Visual Editor without impacting anything outside of the dbt Cloud Development environment.
+- When you're ready to use this mode in a downstream tool, you can run it to materialize it in your data warehouse development schema.
+- Once your model is ready for production and ready to be used by others or orchestrated, commit it and open a PR.
```

Comment:
Best to be specific in case it's not clear to some.
```suggestion
- Once your model is ready for production and ready to be used by others or orchestrated, commit it and open a pull request.
```